src/main/java/io/seqera/tower/cli/responses/TasksView.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2021, Seqera Labs.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0.
+ */
+
+package io.seqera.tower.cli.responses;
+
+import io.seqera.tower.cli.commands.runs.tasks.enums.TaskColumn;
+import io.seqera.tower.cli.utils.TableList;
+import io.seqera.tower.model.ListTasksResponse;
+import io.seqera.tower.model.Task;
+
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+public class TasksView extends Response {
+
+    public final String runId;
+    public final List<TaskColumn> columns;
+    public final ListTasksResponse response;
+
+    public TasksView(String runId, List<TaskColumn> columns, ListTasksResponse response) {
+        this.runId = runId;
+        this.columns = columns;
+        this.response = response;
+    }
+
+    @Override
+    public Object getJSON() {
+        List<List<String>> tasks = new ArrayList<>();
+
+        Objects.requireNonNull(response.getTasks()).forEach(it -> {
+            Task task = it.getTask();
+            List<String> items = columns.stream()
+                    .map(colItem -> colItem.getObject().apply(task) != null ? colItem.getObject().apply(task).toString() : null)
```

Comment:
@jordeu I am sure I can use a `clousure`, `lambda` or `predicate` here along with the table code. Please advise.